proposals/0xxx-enum-cases-as-protocol-witnesses.md

```diff
@@ -0,0 +1,211 @@
+# Enum cases as protocol witnesses
+
+* Proposal: [SE-XXXX](xxxx-enum-cases-as-protocol-witnesses.md)
+* Author: [Suyash Srijan](https://www.github.com/theblixguy)
+* Review Manager: **TBD**
+* Status: **Awaiting Review**
+* Implementation: [apple/swift#28916](https://github.com/apple/swift/pull/28916)
+* Bug: [SR-3170](https://bugs.swift.org/browse/SR-3170)
+* Toolchain: [macOS](https://ci.swift.org/job/swift-PR-toolchain-osx/477/artifact/branch-master/swift-PR-28916-477-osx.tar.gz) & [Linux](https://ci.swift.org/job/swift-PR-toolchain-Linux/346/artifact/branch-master/swift-PR-28916-346-ubuntu16.04.tar.gz)
+
+## Introduction
+
+The aim of this proposal is to lift an existing restriction, which is that enum cases cannot participate in protocol witness matching.
+
+Swift-evolution thread: [Enum cases as protocol witnesses](https://forums.swift.org/t/enum-cases-as-protocol-witnesses/)
```

Comment:
Without a numeric ID, the link will break if the topic is renamed (e.g. by removing `[Pitch]`).

But <https://forums.swift.org/t/foo/32753> will redirect.

```suggestion
Swift-evolution thread: [Enum cases as protocol witnesses](https://forums.swift.org/t/enum-cases-as-protocol-witnesses/32753)
```